var/spack/repos/builtin/packages/py-bx-python/package.py

```diff
@@ -13,10 +13,12 @@ class PyBxPython(PythonPackage):
     homepage = "https://github.com/bxlab/bx-python"
     pypi = "bx-python/bx-python-0.8.8.tar.gz"
 
+    version("0.9.0", sha256="fe545c44d2ea74b239d41e9090618aaf6a859d1a1f64b4a21b133cb602dfdb49")
     version("0.8.8", sha256="ad0808ab19c007e8beebadc31827e0d7560ac0e935f1100fb8cc93607400bb47")
 
-    depends_on("python@2.7:2.8,3.5:", type=("build", "run"))
+    depends_on("python@3.7:3.10", when="@0.8.13:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-cython", type="build")
     depends_on("py-numpy", type=("build", "run"))
-    depends_on("py-six", type=("build", "run"))
+    depends_on("py-six", when="@:0.8.9", type=("build", "run"))
+    depends_on("py-python-lzo@1.14:", type=("build", "run"))
```

Comment:
Might be useful to add a comment saying that this dep isn't listed but is still imported at run-time

Reply:
Clarified that. It was hiding in the `tox.ini` file ...